vtexdocs/helpcenter
# Reverse months order on Announcements sidebar

Reverse months order in Anouncements page, using new navigation.json

#### What problem is this solving?
#43 

#### How should this be manually tested?
- Go to Announcements Page
- Expand Sidebar
- Click a Year
- See order of months   

#### Screenshots or example usage
![image](https://github.com/vtexdocs/helpcenter/assets/55800516/006646ba-5897-4e0f-9f87-61adf4868ff8)

#### Types of changes
